release/_portal_release_tester.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+function get_githash {
+    local branch_name=${1}
+
+	if [[ "${branch_name}" =~ "7.4." ]]
+	then
+		local curl_url="https://api.github.com/repos/brianchandotcom/liferay-portal/commits/${branch_name}"
+	else
+		local curl_url="https://api.github.com/repos/brianchandotcom/liferay-portal-ee/commits/${branch_name}"
+	fi
+
+	local curl_response=$(\
+				curl \
+					${curl_url} \
+					-H "Accept: application/vnd.github.VERSION.sha" \
+					-H "Authorization: Bearer ${GH_TOKEN}" \
+					-s)
+
+    echo "${curl_response}"
+}
+
+function get_test_portal_branch_name {
+    local branch_name=${1}
+
+    if [[ "${branch_name}" =~ "7.4." ]]
+    then
+        echo "master"
+    else
+        echo "${branch_name}"
+    fi
+}
+
+function send_to_ci_test {
+
+	if [ "${SEND_BUILD_TO_TEST_CI}" = "true" ]
+	then
+
+		# Jenkins server details
+		local jenkins_url="http://test-1-1"
+		local job_name="test-portal-release"
```

Comment:
@joshitagaki, it seems these variables are being used just a single time, if it's the case please do not declare them and use their content where it's needed. BChan doesn't like to declare variables used a single time. :wink: